modules/gitops-creating-an-application-by-using-the-argo-cd-dashboard.adoc

```diff
@@ -52,3 +52,7 @@ Directory Recurse:: `checked`
 endif::app[]
 
 . Click *Create* to create your application.
+
+. Open the *Administrator* perspective of the web console and navigate to *Administration* → *Namespaces* in the menu on the left.
```

Comment:
```suggestion
. Open the *Administrator* perspective of the web console and navigate to *Administration* -> *Namespaces* in the menu on the left.
```
Looking at the OpenShift docs, it seems like `->` are the standard characters for linking menu items. I don't see this spelled out in any of our standards, though. It is probably safer to avoid using special characters like the arrow character.